src/calorically_perfect.cpp

```diff
@@ -448,7 +449,8 @@ void CaloricallyPerfectThermoChem::initializeOperators() {
 
   Ht_form_ = new ParBilinearForm(sfes_);
   auto *hmt_blfi = new MassIntegrator(*rho_over_dt_coeff_);
-  auto *hdt_blfi = new DiffusionIntegrator(*thermal_diff_coeff_);
+  // auto *hdt_blfi = new DiffusionIntegrator(*thermal_diff_coeff_);
```

Comment:
something is wrong here, those lines should be

  // auto *hdt_blfi = new DiffusionIntegrator(*thermal_diff_coeff_);                                                                                                            
  auto *hdt_blfi = new DiffusionIntegrator(*thermal_diff_total_coeff_);

where are you seeing this?

Reply:
OH, i see. As long as you are happy with including the multiplier via

  thermal_diff_coeff_ = new GridFunctionCoefficient(&kappa_gf_);
  mult_coeff_ = new GridFunctionCoefficient(sponge_interface_->diff_multiplier);
  gradT_coeff_ = new GradientGridFunctionCoefficient(&Tn_next_gf_);
  thermal_diff_total_coeff_ = new ProductCoefficient(*mult_coeff_, *thermal_diff_coeff_);

then no reason to keep the commented bit.  In general, I kept things commented that either I wasnt sure if you wanted to keep or not or I wasnt sure if what I was doing would cause problems
.  So feel free to delete commented code.